### PR TITLE
add feedback readme

### PR DESCRIPTION
**Pull request by jshawl**

### CODE REVIEW

File: feedback/midcourse-feedback-feedback.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- Talk to students I haven't talked to yet.
+
+## Nick
+
+- Try different teaching approaches more
+- Enjoy delivering lessons more
+- Confidence and comfort in front of the room
+- Cold call more
+- move faster
+- business time
```
Comment (ebirving) [inline feedback/midcourse-feedback-feedback.md]: Make this clearer?

File: feedback/midcourse-feedback-feedback.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Robin
+
+- Cold-call on students more so that there's less responsibility on them to raise their hands when they're feeling lost
+- Approach students more, instead of waiting for students to approach me
+- Share more industry examples
+
+## Andy
+
+- Use more politically correct examples
```
Comment (ebirving) [inline feedback/midcourse-feedback-feedback.md]: "Politically correct" is kind of a loaded term, can be interpreted as insincere. Maybe something like "Use more appropriate humor"

**Comment by ebirving** [comment]
Looks good! Couple of little things inline, also can we standardize capitalization at the beginning of bullet points? Auuuuuuugh I know I'm super annoying.

**Comment by jshawl** [comment]
thanks @ebirving a66a6aa8be11ed8ccd23c6d1b7cbc081b3489aac

**Comment by mattscilipoti** [comment]
:+1: I made minor changes to my steps.  Thanks for help Jesse.

**Pull request merged**